grpc-sys/build.rs

```diff
@@ -379,38 +391,47 @@ fn bindgen_grpc(file_path: &Path) {
 }
 
 // Determine if need to update bindings. Supported platforms do not
-// need to be updated by default unless the UPDATE_BIND is specified.
+// need to be updated by default unless the _gen-bindings feature is specified.
 // Other platforms use bindgen to generate the bindings every time.
 fn config_binding_path() {
     let target = env::var("TARGET").unwrap();
     let file_path: PathBuf = match target.as_str() {
-        "x86_64-unknown-linux-gnu" | "aarch64-unknown-linux-gnu" => {
+        "x86_64-unknown-linux-gnu"
+        | "aarch64-unknown-linux-gnu"
+        | "x86_64-apple-darwin"
+        | "aarch64-apple-darwin" => {
```

Comment:
Why are the bindings the same across this platforms?

Reply:
I guess because they are both unix-like system and share many common designs. The windows bindings is different from linux though. For example, windows use i32 for enum while unix use u32. More to see https://github.com/rust-lang/rust-bindgen/issues/1907

Reply:
I mean, did you found it unconsciously, or did you get the result via bindgen config.

Reply:
I found it when I'm working on https://github.com/BusyJay/tirocks/pull/1.